### DISCUSSION TITLE
我想知道 ss, ssr 是什么编程语言写的。

### DISCUSSION THREAD

**Discussion by nathnial** [XTLS/Xray-core#2235] [Q&A]
因为我找的很多都是python 脚本，所以我很疑惑，难道ss, ssr 都是 python 脚本？

**Comment by yomnxkcs** [comment]
ss最初用的是python，ssr最初用的是c sharp。现在github上有各种语言的实现。其实我想说的是只要能解决问题，用什么语言并不重要。
